src/app/collective-rewards/shared/components/CRWhitepaperLinkNew.tsx

```diff
@@ -0,0 +1,17 @@
+import { CommonComponentProps } from '@/components/commonProps'
+import { ArrowUpRightLightIcon } from '@/components/Icons/ArrowUpRightLightIcon'
+import { Link } from '@/components/Link'
+
+export const CRWhitepaperLink = ({ children, ...props }: CommonComponentProps<HTMLAnchorElement>) => (
+  <Link
+    href="https://rootstockcollective.xyz/pdfs/rewards-whitepaper.pdf"
+    className="no-underline hover:underline gap-1"
+    target="_blank"
+    data-testid="whitepaper-link"
+    rel="noopener noreferrer"
```

Comment:
given that `noopener noreferrer` are a security and privacy measures, should we change the `Link` component to default to use them?

On the other hand, where we are redirecting to our own website, we should not use the noreferrer to not reduce our site analytics for whomever uses it.

See https://www.elegantthemes.com/blog/wordpress/rel-noopener-noreferrer-nofollow for more info